styleguide/derek/global-components/basement/_basement.scss

```diff
@@ -29,6 +29,34 @@
   text-align: center;
 }
 
+.basement-currency,
+.region-select {
+  @include make-sm-column(6);
+  text-align: center;
+}
+
+// Override for bootstrap padding
+div {
+  &.basement-currency {
+    @include make-lg-column-pull(1);
+    padding: 0;
+  }
+}
```

Comment:
Why is this div needed here? Does this not suffice? 
```
.basement-currency {
  @include make-lg-column-pull(1);
  padding: 0;
}
```

When I test it locally it seems fine to me, unless I'm missing something.